pkg/apis/pipeline/v1beta1/taskrun_validation_test.go

```diff
@@ -218,6 +218,29 @@ func TestTaskRunSpec_Invalidate(t *testing.T) {
 		},
 		wantErr: apis.ErrInvalidValue("invalid bundle reference (could not parse reference: invalid reference)", "taskref.bundle"),
 		wc:      enableTektonOCIBundles(t),
+	}, {
+		name: "using debug when apifields stable",
+		spec: v1beta1.TaskRunSpec{
+			TaskRef: &v1beta1.TaskRef{
+				Name: "my-task",
+			},
+			Debug: &v1beta1.TaskRunDebug{
+				Breakpoint: []string{"bReaKdAnCe"},
+			},
+		},
+		wantErr: apis.ErrDisallowedFields("debug"),
+	}, {
+		name: "invalid breakpoint",
+		spec: v1beta1.TaskRunSpec{
+			TaskRef: &v1beta1.TaskRef{
+				Name: "my-task",
+			},
+			Debug: &v1beta1.TaskRunDebug{
+				Breakpoint: []string{"breakito"},
+			},
+		},
+		wantErr: apis.ErrInvalidValue("breakito is not a valid breakpoint", "debug.breakpoint"),
```

Comment:
This error message doesn't give me a lot of information about why it's not valid, could you add more context?

Reply:
Added the available valid breakpoints to the error.